docs/quickstart.md

```diff
@@ -0,0 +1,193 @@
+# Quickstart
+
+Curious to see CNWAN Operator in action but feeling lazy about learning
+how it does stuff? Follow this guide!
+
+## Requirements
+
+To run this, make sure you have:
+
+* Access to a Kubernetes cluster running at least version `1.11.3`
+  * [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/)
+  is also fine.
+* [Kubectl 1.11.3+](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* A Project in [Google Cloud](https://console.cloud.google.com/) with
+[Service Directory](https://cloud.google.com/service-directory) enabled
+* A [Google Cloud Service Account](https://cloud.google.com/iam/docs/service-accounts)
```

Comment:
We need to specify the minimum roles the service accounts needs. Is it Service Directory Admin (`roles/servicedirectory.admin`) ?

Reply:
Good catch! Most probably not, but I will look into it, thank you!

Reply:
I specified the roles needed. Thank you!